README.md

```diff
@@ -70,41 +78,44 @@ Sett prosjekt (miljø) du skal migrere:
 gcloud config set project <PROJECT_ID>
 ```
 
-# Før migrering må man grante usage på schema og på tabeller
-```GRANT USAGE on SCHEMA public to "cloudsqliamserviceaccount";```
-
-```GRANT INSERT ON ALL TABLES IN SCHEMA public TO "cloudsqliamserviceaccount"; ```
+Instansen er nå klar til å koble seg på databaser og starte migreringen.
 
-#### OBS! Er nye tabeller lagt til må de også få spesifikk GRANT kjørt på seg. 
-Dette må da gjøres via deploy(anbefalt) eller via superbruker da utviklere via iam tilgang
-ikke har lov til å grante via ` nais postgres proxy`.
-Eksempel: 
-`kubectl get secret google-sql-etterlatte-vilkaarsvurdering -o json | jq '.data | map_values(@base64d)'`
-og logg inn som rot.(anbefales ikke)
+## Forarbeid databaser
+Videre guide tar utgangspunkt i at skjemadefinisjoner (DDL) allerede er opprettet i 
+databasen det migreres til. For å hente ut dette kan man kjøre:
 
-## Legg inn tabeller uten flyway tabeller i app man ønsker å migrere til
-Her er det veldig lurt å tenke på om man vi vil legge på indeksene i etterkant da 
-insert med hele tabellen sammen med indeksering kan ta kjempelang tid.
-
-For å finne skjemadefinisjonene man vil legge kan man se dette ved å kjøre feks:
-`pg_dump -h localhost -p 5432 -U dinbruker@nav.no(eller lignende) -d vilkaarsvurdering --schema-only`
+```shell
+pg_dump -h localhost -p 5432 -U <BRUKER> -d <DATABASE> --schema-only --exclude-table-data=flyway_schema_history
+```
 
+Det kan være lurt å tenke på om man vi vil legge på indeksene i etterkant da
+insert med hele tabellen sammen med indeksering kan ta lang tid. 
```

Comment:
"sjekk indekstørrelse opp mot tabellstørrelse for å avgjøre dette evt en test i dev/lokalt"